users/333fred/rules.mk

```
@@ -7,3 +7,4 @@ endif
 TAP_DANCE_ENABLE=yes
 NKRO_ENABLE = yes
 PERMISSIVE_HOLD = yes
+EXTRAFLAGS += -flto
```

```suggestion
LTO_ENABLE = yes
```
This also turns off action macros and action functions, as they tend to error when compiling with lto enabled.

---

Thanks, I knew there was a flag for this but couldn't find it.